CONTRIBUTING.md

```diff
@@ -2,15 +2,15 @@
 Welcome! Thanks for checking out the new Expensify app and for taking the time to contribute!
 
 ## Getting Started
-If you would like to become an Expensify contributor, the first step is to read this document in its entirety. The second step is to review the README guidelines [here](https://github.com/Expensify/App/blob/main/README.md) for a general overview of the code repository (i.e. how to run the app locally, testing, storage, etc). Please read both documents before asking questions, as it may be covered within the documentation.
+If you would like to become an Expensify contributor, the first step is to read this document in its entirety. The second step is to review the README guidelines [here](https://github.com/Expensify/App/blob/main/README.md) for a general overview of the code repository (i.e. how to run the app locally, testing, storage, our app philosophy, etc). Please read both documents before asking questions, as it may be covered within the documentation.
```

Comment:
`app philosophy` is kinda ambiguous here. I'd just say `best practices` maybe (unless you wanted something more general).

Reply:
ahhh I like this more! App philosophy seems like it encompasses those other facets.

Reply:
Nice, agreed!